--- libreport-2.1.5/src/plugins/ureport.c.orig	2013-07-25 15:39:39.289878178 +0200
+++ libreport-2.1.5/src/plugins/ureport.c	2013-07-25 15:42:58.916536467 +0200
@@ -476,7 +476,7 @@
         {
             log(_("This problem has already been reported."));
             if (response->message)
-                log(response->message);
+                log("%s", response->message);
 
             ret = EXIT_STOP_EVENT_RUN;
         }
--- libreport-2.0.16/src/gui-wizard-gtk/wizard.c~	2012-10-10 18:11:04.000000000 +0200
+++ libreport-2.0.16/src/gui-wizard-gtk/wizard.c	2012-10-15 12:55:58.123197354 +0200
@@ -281,7 +281,7 @@
         GTK_DIALOG_MODAL | GTK_DIALOG_DESTROY_WITH_PARENT,
         GTK_MESSAGE_WARNING,
         GTK_BUTTONS_CLOSE,
-        message);
+        "%s", message);
 
     gtk_window_set_transient_for(GTK_WINDOW(wrong_settings), GTK_WINDOW(g_wnd_assistant));
     gtk_message_dialog_set_markup(GTK_MESSAGE_DIALOG(wrong_settings),
@@ -2753,7 +2753,7 @@
                 GTK_DIALOG_MODAL | GTK_DIALOG_DESTROY_WITH_PARENT,
                 GTK_MESSAGE_WARNING,
                 GTK_BUTTONS_CLOSE,
-                message);
+                "%s", message);
             free(message);
             gtk_window_set_transient_for(GTK_WINDOW(dlg), GTK_WINDOW(g_wnd_assistant));
             gtk_dialog_run(GTK_DIALOG(dlg));
