--- libreport-2.7.0/src/plugins/reporter-ureport.c~	2016-04-07 18:24:05.000000000 +0300
+++ libreport-2.7.0/src/plugins/reporter-ureport.c	2016-04-27 10:31:21.566593068 +0300
@@ -312,7 +312,7 @@
         {
             log(_("This problem has already been reported."));
             if (response->urr_message)
-                log(response->urr_message);
+                log("%s", response->urr_message);
 
             ret = EXIT_STOP_EVENT_RUN;
         }
--- libreport-2.7.0/src/gui-wizard-gtk/wizard.c~	2016-04-07 18:24:05.000000000 +0300
+++ libreport-2.7.0/src/gui-wizard-gtk/wizard.c	2016-04-27 10:39:33.488656568 +0300
@@ -3352,7 +3352,7 @@
                 GTK_DIALOG_MODAL | GTK_DIALOG_DESTROY_WITH_PARENT,
                 GTK_MESSAGE_WARNING,
                 GTK_BUTTONS_CLOSE,
-                message);
+                "%s", message);
             free(message);
             gtk_window_set_transient_for(GTK_WINDOW(dlg), GTK_WINDOW(g_wnd_assistant));
             gtk_dialog_run(GTK_DIALOG(dlg));
--- libreport-2.7.0/src/plugins/reporter-rhtsupport.c~	2016-04-07 18:24:05.000000000 +0300
+++ libreport-2.7.0/src/plugins/reporter-rhtsupport.c	2016-04-27 10:35:22.815716392 +0300
@@ -288,7 +288,7 @@
         ureport_server_response_save_in_dump_dir(resp, dump_dir_name, conf);
 
         if (resp->urr_message)
-            log(resp->urr_message);
+            log("%s", resp->urr_message);
     }
     else if (g_verbose > 2)
         error_msg(_("Server responded with an error: '%s'"), resp->urr_value);
