--- libreport-2.1.5/src/plugins/ureport.c.orig	2013-07-25 15:39:39.289878178 +0200
+++ libreport-2.1.5/src/plugins/ureport.c	2013-07-25 15:42:58.916536467 +0200
@@ -476,7 +476,7 @@
         {
             log(_("This problem has already been reported."));
             if (response->message)
-                log(response->message);
+                log("%s", response->message);
 
             ret = EXIT_STOP_EVENT_RUN;
         }
--- libreport-2.2.2/src/gui-wizard-gtk/wizard.c.orig	2014-04-20 22:07:04.000000000 +0200
+++ libreport-2.2.2/src/gui-wizard-gtk/wizard.c	2014-06-14 17:08:36.638269706 +0200
@@ -1935,7 +1935,7 @@
         GTK_DIALOG_MODAL | GTK_DIALOG_DESTROY_WITH_PARENT,
         GTK_MESSAGE_WARNING,
         GTK_BUTTONS_YES_NO,
-        message);
+        "%s", message);
 
     gtk_window_set_transient_for(GTK_WINDOW(old_private_group), GTK_WINDOW(g_wnd_assistant));
     gtk_message_dialog_set_markup(GTK_MESSAGE_DIALOG(old_private_group),
@@ -3118,7 +3118,7 @@
                 GTK_DIALOG_MODAL | GTK_DIALOG_DESTROY_WITH_PARENT,
                 GTK_MESSAGE_WARNING,
                 GTK_BUTTONS_CLOSE,
-                message);
+                "%s", message);
             free(message);
             gtk_window_set_transient_for(GTK_WINDOW(dlg), GTK_WINDOW(g_wnd_assistant));
             gtk_dialog_run(GTK_DIALOG(dlg));
