--- libreport-2.6.0/tests/xfuncs.at.orig	2015-06-08 17:13:44.000000000 +0200
+++ libreport-2.6.0/tests/xfuncs.at	2015-06-27 17:10:28.975037433 +0200
@@ -88,7 +88,7 @@
 
         char buf[sizeof(unsigned long) * 3 + 1];
 
-        snprintf(buf, sizeof(buf), "%lu", 1LU + UINT_MAX);
+        snprintf(buf, sizeof(buf), "%llu", 1LLU + UINT_MAX);
         assert(try_atou(buf, &uint_value) != 0 || !"Above UINT_MAX");
         assert(uint_value == 12345);
 
@@ -115,11 +115,11 @@
 
         char buf[sizeof(long) * 3 + 1];
 
-        snprintf(buf, sizeof(buf), "%ld", 1L + INT_MAX);
+        snprintf(buf, sizeof(buf), "%lld", 1LL + INT_MAX);
         assert(try_atoi(buf, &int_value) != 0 || !"Parse INT_MAX+1");
         assert(int_value == 12345 || !"Above INT_MAX");
 
-        snprintf(buf, sizeof(buf), "%ld", -1L + INT_MIN);
+        snprintf(buf, sizeof(buf), "%lld", -1LL + INT_MIN);
         assert(try_atoi(buf, &int_value) != 0 || !"Parse INT_MIN-1");
         assert(int_value == 12345 || !"Belove INT_MIN");
 
@@ -144,7 +144,7 @@
 
         char buf[sizeof(long) * 3 + 1];
 
-        snprintf(buf, sizeof(buf), "%ld", 1L + INT_MAX);
+        snprintf(buf, sizeof(buf), "%lld", 1LL + INT_MAX);
         assert(try_atoi_positive(buf, &positive_value) != 0);
         assert(positive_value == 12345 || !"Above INT_MAX");
 
